Update events.md (#1280)
xsf/xmpp.org: content/community/events.md

    - "/community/events.html"
---

## Meetups

* [Berlin](https://xmpp-meetup.in-berlin.de/)
* [Dresden](https://wiki.xmpp.org/web/Meetups/Dresden)
* [XMPP Italian happy hour](https://mobilizon.it/events/207032cc-b7bf-48c2-9394-f5cce684608f)
* [XMPP Office Hours](https://wiki.xmpp.org/web/XMPP_Office_Hours) are offered on a weekly manner and offer everyone to talk, present and discuss their XMPP topic, project or development. Previous talks can be found in our [YouTube channel](https://www.youtube.com/channel/UCf3Kq2ElJDFQhYDdjn18RuA)!

### Upcoming Events

> Events come and go at a speed that is sometimes hard to keep track of. Do you know of an event not listed here? Did you spot out-of-date information? Can you otherwise improve on this page? Please, [submit a pull request!](https://github.com/xsf/xmpp.org/edit/master/content/community/events.md) Sprints are contributor events that are being organized by members of the community, to improve the ecosystem. You can find more detailed information about sprints on the [wikipage](https://wiki.xmpp.org/web/Sprints). You can get a list of past conferences [on the wiki](https://wiki.xmpp.org/web/Category:Events).

Reviewer: ```suggestion
Events come and go at a speed that is sometimes hard to keep track of. Do you know of an event not listed here? Do you spot out-of-date information? Can you improve this page? Please, [submit a pull request!](https://github.com/xsf/xmpp.org/edit/master/content/community/events.md). Sprints are contributor events organized by members of the community. They focus to improve the XMPP ecosystem. You can find detailed information about sprints on the [wiki page](https://wiki.xmpp.org/web/Sprints), as well as a list of [past conferences](https://wiki.xmpp.org/web/Category:Events).
```
